app/frontend/src/features/profile/view/ProfilePage.test.tsx

```diff
@@ -95,6 +108,26 @@ describe("Profile page", () => {
         screen.queryByRole("button", { name: MORE_PROFILE_ACTIONS_A11Y_TEXT })
       ).not.toBeInTheDocument();
     });
+
+    describe("and a tab is opened", () => {
+      it("updates the url with the chosen tab value", async () => {
+        renderProfilePage();
+
+        expect(testLocation.pathname).toBe("/user");
+
+        await act(async () => {
+          userEvent.click(await screen.findByText(SECTION_LABELS.home));
+        });
```

Comment:
```suggestion
        userEvent.click(await screen.findByText(SECTION_LABELS.home));
```
`userEvent` already wrap things in `act` with all of its methods so we don't need to explicitly do it.